WordPress/src/test/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadViewModelTest.kt

```diff
@@ -34,13 +164,56 @@ class BackupDownloadViewModelTest {
         assertThat(toolbarStates.size).isEqualTo(0)
     }
 
+    @Test
+    fun `backupDownloadState is writtenToBundle`() {
+        viewModel.start(savedInstanceState = null)
+
+        viewModel.writeToBundle(savedInstanceState)
+        verify(savedInstanceState)
+                .putParcelable(any(), argThat { this is BackupDownloadState })
+    }
+
+    @Test
+    fun `when setToolbarState is invoked, toolbar state is updated`() {
+        val toolbarStates = initObservers().toolbarState
+
+        viewModel.start(null)
+        viewModel.setToolbarState(DetailsToolbarState())
+        assertThat(toolbarStates.last()).isInstanceOf(DetailsToolbarState::class.java)
+
+        viewModel.setToolbarState(ProgressToolbarState())
+        assertThat(toolbarStates.last()).isInstanceOf(ProgressToolbarState::class.java)
+
+        viewModel.setToolbarState(CompleteToolbarState())
+        assertThat(toolbarStates.last()).isInstanceOf(CompleteToolbarState::class.java)
+    }
+
+    @Test
+    fun `step index is restored`() {
```

Comment:
Minor (🔍): Consider adding empty lines to split the `Arrange`, `Act` and `Assert`.

Reply:
I split these out based on your earlier suggestion. :)